### DISCUSSION TITLE
Should we start using Github wiki?

### DISCUSSION THREAD

**Discussion by jonyo** [geodesicsolutions-community/geocore-community#34] [Q&A] [answered]
I was thinking, maybe we should start using the github wiki.

* We would keep the old wiki in place at https://geodesicsolutions.org/wiki (but perhaps look into ways to import pages easily)
* It would allow easier "onboarding" for people to contribute if they want. 
* Maybe more people familiar with how github wiki syntax works.
* No worrying about backing up or if the geodesicsolutions.org site goes down...  Since it is run by github, they are stable esp. compared to my website that has no uptime monitoring or anything like that.
* Would improve "struck by a bus" proof-ness of the community and project.  (The concept meaning, how resilient is the community / project if a single person is struck by a bus and ends up in a hospital for a long time)

Downside:  My only concern is the "single source of truth" - it might not be good to have 2 wikis. We can mitigate that by calling the other one the "legacy" wiki or something and lock it down to no more edits... But it may still make things more confusing.

What do you think, good or bad idea to start using github wiki? Or maybe use it but only for "github related" info?

**Comment by vicos59** [comment, marked as answer]
Your logic is very sound. I am always in favor of keeping as much as possible in one place, and after what happened with the company going under, I don't think anyone could argue against bus-proof.
